components/02-elements/form/fieldset/_fieldset.scss

```diff
@@ -0,0 +1,19 @@
+$fieldset__border            : none !default;
+$fieldset__margin            : 0 !default;
+$fieldset__padding           : 0 !default;
+$fieldset__legend-margin     : 0 0 25px !default;
+$fieldset__legend-font-size  : $font-size-large !default;
+$fieldset__legend-line-height: 1.2 !default;
+
+.fieldset {
+    border: $fieldset__border;
+    padding: $fieldset__padding;
+    margin: $fieldset__margin;
+}
+.legend {
```

Comment:
You have different selectors in CSS and HTML. This should be a BEM element.

Reply:
I forgot to change here. thx.